UnitTests/PayPalUI_Tests/PayPalButton_Tests.swift

```diff
@@ -13,9 +13,9 @@ class PayPalButton_Tests: XCTestCase {
         XCTAssertEqual(payPalButton.containerView.backgroundColor, PaymentButtonColor.gold.color)
```

Comment:
Do we also want to update this test to match the updates you made for the other button tests? To include label, insets, collapsed, etc?

Reply:
done!